sites/bizbash.com/config/navigation.js

```diff
@@ -137,7 +138,7 @@ module.exports = {
       items: [
         { href: '/gathergeeks', label: 'GatherGeeks Podcast' },
         { href: '/webinars', label: 'Webinars' },
-        { href: '/page/our-events', label: 'Events & Awards' },
+        { href: '/page/our-events', label: 'Events' },
```

Comment:
Might want to just add the new awards page here too as it's being split in the nav already.